x-pack/plugins/triggers_actions_ui/public/application/hooks/use_load_rule_alerts_aggregations.ts

```diff
@@ -251,7 +261,11 @@ export async function fetchRuleAlertsAggByTimeRange({
     return {
       active,
       recovered,
-      alertsChartData,
+      alertsChartData: [
+        ...alertsChartData.filter((acd) => acd.status === 'active'),
+        ...alertsChartData.filter((acd) => acd.status === 'recovered'),
+        ...alertsChartData.filter((acd) => acd.status === 'total'),
+      ],
```

Comment:
It's a kind of an expensive operation. `alertsChartData` > 30 * 3 = < 90 filter iterations.

I suggest keeping the comment too. 
```suggestion
// We need the chart data order to be: `active` first, then `recovered`, and last `total`. And it's happened to be in the same order alphabetically. 
      alertsChartData: sortBy(alertsChartData, 'status'),
```

If you agree with the suggestion, we need to `import sortBy from 'lodash/sortBy';`

Reply:
we will have two sort on two attributes (time and status) to make sure it is still working

Reply:
I am going to keep it the way it is, if we really need to work on performance, I will prefer do the work in elastic search query. I think we can use some scripting if we really want to